pennylane/operation.py

```diff
@@ -962,15 +962,6 @@ def kraus_matrices(self):
         return self._kraus_matrices(*self.parameters)
 
     def __init__(self, *params, wires=None, do_queue=True):
```

Comment:
@ixfoduap now that this init method doesn't do anything except call `super()`, you can delete it :)

Reply:
Thanks! This was something I believe Aroosa did back in the day, not sure why we didn't catch that it wasn't needed

Reply:
Oh, no this is a change from this PR (due to the errors being deleted, the `__init__` method is no longer needed)